eng/InstallRuntimes.proj

```diff
@@ -20,9 +26,14 @@
   -->
 
   <PropertyGroup>
-    <DailyTest Condition="'$(DailyTest)' == ''">false</DailyTest>
     <BuildArch Condition="'$(BuildArch)' == ''">$(Platform)</BuildArch>
     <BuildArch Condition="'$(BuildArch)' == ''">x64</BuildArch>
+    <PrivateBuildTesting>false</PrivateBuildTesting>
+    <PrivateBuildTesting Condition="'$(PrivateBuildPath)' != ''">true</PrivateBuildTesting>
+    <InternalReleaseTesting>false</InternalReleaseTesting>
+    <InternalReleaseTesting Condition="'$(DotnetRuntimeVersion)' != '' and '$(DotnetRuntimeVersion)' != 'default'">true</InternalReleaseTesting>
```

Comment:
The comment above and the validation in build.ps1 don't seem to allow default as a parameter value. We should only have one and let it be the default.